### DISCUSSION TITLE
to add recursive in filter panel

### DISCUSSION THREAD

**Discussion by Aliaksandr3by** [doublecmd/doublecmd#1431] [Suggestions]
to add recursive (searching in subfolder) in filter panel without being flattened (ctrl+b)

![image](https://github.com/doublecmd/doublecmd/assets/11432724/d33e4a6d-4688-42d9-9b50-642e98ea94b6)

**Comment by alexx2000** [comment]
Normal search exists for such task. Quick search finds files only in the panel file list.

**Comment by Aliaksandr3by** [comment]
it is exist many program for this task e.g. "everything".
but I think is good idea to add the function. If it is easy to do.
If it is many work then bad Idea to do it.

**Comment by alexx2000** [comment]
It is a totally different function.
